src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -170,6 +312,31 @@ object ConductrPlugin extends AutoPlugin {
   def sandboxStop(): Unit =
     Process(Seq("sandbox", "stop")).!
 
+  /**
+   * A convenience function that waits on ConductR to become available.
+   *
+   * @param duration the max time to wait.
+   */
+  def waitForConductr(implicit duration: FiniteDuration): Unit = {
+    print("Waiting for ConductR to start")
+
+    val deadline = duration.fromNow
+
+    @tailrec
+    def loop(deadline: Deadline): Unit = {
+      print(".")
+      if (deadline.isOverdue())
+        throw new TimeoutException(s"ConductR has not been started within ${duration.toSeconds} seconds!")
```

Comment:
Not sure how the exception will be displayed to the user (doesn't one have to type `last`?), but to me it seems a log message is a nicer way of telling the user what went wrong (that's what I'm doing in Lagom, see [here](https://github.com/lagom/lagom/blob/master/dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/run/Servers.scala#L158) for an example)